[Feat] Mypage_Modify HTML

kb4954 opened the pull request:
**Check List :memo:**
- [x] Pull Request 제목 : [유형] 변경 사항 요약
- [x] 설명 & 이슈 번호 작성

**설명 (필수)**

마이페이지 내 정보수정 및 계정삭제 페이지입니다.
출력만 가능하고 js 추후 추가예정입니다.

1. #{15}

2. 변경 사항
비밀번호 input box 추가
클릭시 border 변화 추가

4. 스크린샷 (선택)
![스크린샷(1)](https://github.com/user-attachments/assets/35ef9937-f337-40e9-8798-0c72e27f78bd)

<br>

---

**review 참고 사항 (선택)**

추가로 알릴 내용이 있다면 여기에 작성해주세요.

1. 변경할 비밀번호를 입력해주세요 멘트 추후 input 박스 내 추가할 예정

2.

kb4954 committed:
mypage_Modify

마이페이지 내 정보수정 및 계정삭제 페이지입니다.
출력만 가능하고 js 아직 추후 추가예정입니다.

kb4954 committed:
feat:1:1cs와 디자인통일

1:1게시판과 디자인 전체적으로 통일함.
변경하기 버튼 클릭시 팝업
계정삭제 버튼 클릭시 팝업 및 alert
헤더푸터 페이지네이터 추가

null4848 commented on my_Modify/css/popup.css:
```diff
@@ -0,0 +1,90 @@
+font-size: 13px;
+border-radius: 7px;
+width: 80px;
+height: 28px;
```
height 45px 정도로
정보수정/계정삭제 부분 버튼이랑 통일해주셔도 좋을 것 같습니다.

null4848 commented on my_Modify/4_Modify_headerfooter.html:
```diff
@@ -0,0 +1,345 @@
+                            <input type="text" placeholder="변경할 비밀번호를 입력하세요" >
+                        </div>
+                        <div class="re_button">
+                            <button class="sui_button pwBtn">비밀번호 변경하기</button>
```
버튼 색 통일해주세요!

null4848 commented on my_Modify/css/4My_Modify.css:
```diff
@@ -0,0 +1,255 @@
+}    
+.field_description{
+    margin-bottom: 7px;
+    color: #A0A0A0;
```
색 너무 연해요!

null4848 commented on my_Modify/css/4My_Modify.css:
```diff
@@ -0,0 +1,255 @@
+}
+.field_description_delete{
+    padding: 0px 0px 13px;
+    color: #999;
```
색 너무 연해요!

null4848 reviewed (commented):
field_description_delete 부분이랑 field_description 부분 색 통일해주시면 좋을 것 같아요!

kb4954 committed:
feat: 버튼색, 설명문글자색 수정

.popup-content .confirmPwBtn
height:45px로 변경

설명글 회색부분
둘다 #636363으로 글자색 변경
비밀번호 변경 버튼색 변경

kb4954 committed:
feat:카테고리탭 수정,css 전체 수정
